gulpfile.js

```diff
@@ -21,15 +41,8 @@ gulp.task("copy", function(cb) {
     });
 });
 
-gulp.task("clean", function(cb) {
-    del("./coverage")
-    .then(function(files) {
-        if(files.length > 0) {
-            gutil.log("Deleted %s", files);
-        }
-        cb();
-    });
+gulp.task("dev", ["clear", "clean"], function(cb) {
```

Comment:
Probably an idea for a future ticket, but ideally it would be neat if it wasn't necessary to clean prior to building :)

Reply:
See my answer about logging ^